docs/about.md

```diff
@@ -0,0 +1,9 @@
+---
+id: about
+title: About us
+sidebar_label: About us
+---
+
+Unmock is created by a small team that co-owns, with some great investors, a company called Meeshkan. We build products that help people develop software that relies on external services, like REST APIs, databases, and IoT connections.
```

Comment:
The first sentence is a bit confusing to me. Does `unmock` owns `Meeshkan` or vice versa (maybe that's also a topic for internal conversation :smile: )

Reply:
Hmm to me it quite clearly says that the team that created unmock also co-owns Meeshkan. But this is all copied from the existing docs so can be refined later.

Reply:
Ah yeah, needed to read that with a fresh set of eyes.